frontend/config-assistant/src/components/toolbar/TopMenuBar.ts

```diff
@@ -240,11 +230,6 @@ export class TopMenuBar {
       {
         separator: true,
```

Comment:
Please remove the separator in the menu bar before the search bar.
When doing that, you can also remove the share button and the separator before that, because we will never use that
![image](https://github.com/PaulBredl/config-assistant/assets/34026908/89623b9d-01aa-4c9e-89fb-fbacd86e5e5b)

Reply:
> Please remove the separator in the menu bar before the search bar. When doing that, you can also remove the share button and the separator before that, because we will never use that ![image](https://user-images.githubusercontent.com/34026908/271562365-89623b9d-01aa-4c9e-89fb-fbacd86e5e5b.png)

@PaulBredl  did it